### PR TITLE
solution

### PR DESCRIPTION
**Pull request by DatsOlena**
- [DEMO LINK](https://DatsOlena.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://DatsOlena.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <div class="container">
+        <a href="#home" class="logo"><img src="./images/logo.png" alt="logo"></a>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:15]:
```suggestion
        <a href="#home" class="logo">
          <img src="./images/logo.png" alt="logo">
        </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,85 @@
...
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+a {
+  text-decoration: none;
+  transition: all 0.3s ease-out;
+  outline: none;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:15]: use class selectors

File: src/style.css [hunk 3]
```diff
@@ -1 +1,85 @@
...
+
+.header {
+  padding: 10px 50px;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:19]:
```suggestion
.header {
  padding: 0 50px;
}
```

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Screenshot 2022-07-29 at 17 40 44

* lets leave only horizontal paddings for header and make it vertically stretched by header content
* header has box-shadow in design - lets add it
* lets remove left-margin from first link
Screenshot 2022-07-29 at 17 45 56

**Pull request closed without merging**